### PR TITLE
User group for Cluster API release team

### PR DESCRIPTION
**Pull request by ykakarap**
This PR adds a slack user group for the Cluster API release team.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]
/lgtm

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

File: communication/slack-config/usergroups.yaml [hunk 1]
```diff
@@ -224,3 +224,21 @@ usergroups:
       - rtaniwa
       - schultzp2020
       - yangcao77
+
+  - name: capi-release-team
```
Comment (CecileRobertMichon) [inline communication/slack-config/usergroups.yaml:228]: nit: I noticed the github team is called `cluster-api-release-team`, would it make sense to call them both the same so they're easier to find?

**Review by CecileRobertMichon — COMMENTED** [review]
/lgtm

**Comment by sbueringer** [comment]
/lgtm

Didn't verify the IDs, but the slack IDs are fine based on our release team doc in CAPI.

**Comment by sbueringer** [comment]
/assign @jdumars 
:)

**Comment by jdumars** [comment]
/approve

Yay team for organizing this effort!

**Pull request merged**